docs/getting-started/security-ui.asciidoc

```diff
@@ -22,204 +23,69 @@ image::images/search-bar.png[]
-
-[float]
-[[cases-ui]]
-=== Cases page
-
-The Cases page is used to open and track security issues directly in the {security-app}. Refer to <<cases-overview, Cases>> for more information.
-
-[role="screenshot"]
-image::cases/images/cases-home-page.png[Shows the Cases page]
-
-[float]
-[[endpoints-page]]
-=== Endpoints page
-
-The Endpoints page allows you to view and manage hosts running {endpoint-cloud-sec}. Refer to <<admin-page-ov, Endpoints>> for more information.
-
-[role="screenshot"]
-image::management/admin/images/endpoints-pg.png[Shows the Endpoints page]
-
-[float]
-[[policies-page]]
-=== Policies page
-
-The Policies page lists all of the integration policies configured for {endpoint-cloud-sec}.  Refer to <<policies-page-ov, Policies>> for more information.
-
-[role="screenshot"]
-image::management/admin/images/policy-list.png[Policies page]
-
-[float]
-[[trusted-apps-page]]
-=== Trusted applications page
-
-The Trusted applications page allows you to add Windows, macOS, and Linux applications that should be trusted. Refer to <<trusted-apps-ov, Trusted applications>> for more information.
-
-[role="screenshot"]
-image::management/admin/images/trusted-apps-list.png[Shows the Trusted applications page]
-
-[float]
-[[event-filters-page]]
-=== Event filters page
-
-The Event filters page allows you to filter endpoint events that you do not need or want stored in {es}. Refer to <<event-filters, Event filters>> for more information.
-
-[role="screenshot"]
-image::management/admin/images/event-filters-list.png[Shows the Event filters page]
+image::images/getting-started-pg.png[Shows the Get started page]
 
-[float]
-[[host-isolation-exceptions-page]]
-=== Host isolation exceptions page
+*Manage*: Expand this section to manage these frequently used tools in the {security-app}:
 
-The Host isolation exceptions page allows you to specify IP addresses that allow communication with isolated hosts, even when blocked from the rest of your network. Refer to <<host-isolation-exceptions, Host isolation exceptions>> for more information.
+* <<rules-ui-create, *Rules*>>: Create and manage rules to monitor suspicious events.
+* <<detections-ui-exceptions, *Exception lists*>>: View and manage all rule exceptions.
+* <<admin-page-ov, *Endpoints*>>. View and manage hosts running {endpoint-cloud-sec}.
+* <<policies-page-ov, *Policies*>>: View and manage the integration policies configured for {endpoint-cloud-sec}.
+* <<trusted-apps-ov, *Trusted applications*>>: View and manage Windows, macOS, and Linux applications that should be trusted.
+* <<event-filters, *Event filters*>> View and manage event filters, which allow you to filter endpoint events you don't need to want stored in {es}.
+* <<host-isolation-exceptions, *Host isolation exceptions*>>: View and manage host isolation exceptions, which allow you to specify IP addresses that allow communication with isolated hosts, even when blocked from the rest of your network.
```

Comment:
```suggestion
* <<host-isolation-exceptions, *Host isolation exceptions*>>: View and manage host isolation exceptions, which specify IP addresses that can communicate with your hosts even when those hosts are isolated.
```

Reply:
I like @benironside's edit above, but think it's still useful to say "blocked from..." as a way of explaining what "host isolation" means, instead of just repeating it. But we could prob shorten it:
```suggestion
* <<host-isolation-exceptions, *Host isolation exceptions*>>: View and manage host isolation exceptions, which specify IP addresses that can communicate with your hosts even when those hosts are blocked from your network.
```